ember-app/app/controllers/application.js

```diff
@@ -24,6 +24,25 @@ var ApplicationController = Ember.Controller.extend(
 
   //Fix the need to delay event subscriptions
   subscribeDisabled: true,
+
+  //Browser Session Checker
+  boardSyncing: Ember.inject.service(),
+  checkBrowserSession: function(){
+    var lastFocus = this.get('browser-session.lastFocus');
+    var _self = this;
+    if(lastFocus >= _self.browserCheckInterval){
+      var since = new Date(new Date().getTime() - _self.browserCheckInterval);
+      this.validateCredentials().success((response)=>{
+        _self.get('boardSyncing').syncIssues(_self.get('model.board'), {since: since.toISOString()});
+      }).fail((error)=>{
```

Comment:
Rather than separately/explicitly check if the user is logged in here, it seems like we should just handle a 403 when we attempt to fetch latest. More generally, could we use [`jQuery.ajaxSetup`](https://api.jquery.com/jquery.ajaxsetup/) (or a more Ember-y equivalent?) to do something like:

```javascript
$.ajaxSetup({
  statusCode: {
    403: function() {
      alert( "Access denied! Do you need to [login|grant private access]?" );
    },
    500: function() {
      alert( "Oh noes, something went wrong! Try again?" );
    }
  }
});
```

(With better UX than alerts, of course.)